charts/mediawiki/templates/hpa-web.yaml

```diff
@@ -0,0 +1,21 @@
+{{- if .Values.web.autoscaling.enabled -}}
+apiVersion: autoscaling/v2
+kind: HorizontalPodAutoscaler
+metadata:
+  name: mediawiki-139-app-web
+  namespace: default
+spec:
+  maxReplicas: {{ .Values.web.autoscaling.maxReplicas }}
+  minReplicas: {{ .Values.web.autoscaling.minReplicas }}
+  metrics:
+  - resource:
+      name: cpu
+      target:
+        averageUtilization: 80
```

Comment:
Question: where is this number coming from? Just a guess or is it backed by anything? I'm not opposed to value, but would like to understand better where it's coming from.

Also: would it make sense to have it defined in `values`?

Reply:
I was a little vague in the PR
> 80% cpu was chosen as it is the default for horizontal pod autoscaling.

This is the default kubernetes has internally for doing this autoscaling see: [this bit of the docs](https://kubernetes.io/docs/reference/kubernetes-api/workload-resources/horizontal-pod-autoscaler-v2/#:~:text=If%20not%20set%2C%20the%20default%20metric%20will%20be%20set%20to%2080%25%20average%20CPU%20utilization).
.

It was chosen pretty randomly but I think given our issues were seeing cpu totally saturate this should be sufficient. I currently think we won't need to adjust this and therefore would suggest we make it a variable the first time we need to.